Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -537,6 +538,24 @@ define([
             }
         },
 
+        /**
+         * Gets the request options.  This function should not be called before {@link UrlTemplateImageryProvider#ready} returns true.
+         * @memberof UrlTemplateImageryProvider.prototype
+         * @type {Object}
+         * @readonly
+         * @default undefined
+         */
+        requestOptions : {
+            get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this.ready) {
```

Comment:
Not sure this is needed for this property, since it's set in the constructor, right?